iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -810,14 +853,22 @@ private TSFetchResultsResp getTSFetchResultsResp(TS_StatusCode code, String msg)
   }
 
   public void handleClientExit() throws TException {
+    closeClusterService();
     closeOperation(null);
     closeSession(null);
   }
 
+  /**
+   * Close cluster service
+   */
+  public void closeClusterService() {
```

Comment:
Why add this empty method?

Reply:
In cluster module, when close jdbc service, it needs to close some cluster service.

Reply:
if it is only used in cluster module, why not only add it in `TSClusterServiceImpl`?

Reply:
It needs to override `handleClientExit()` and invoke duplicated codes, so I origin choose to add it in TSServiceImpl.  But `handleClientExit()` has only two line code, and it indeed more clear to override it in cluster module, so accept your advice. fixed